submissions/vovan-zt/app_friends/style.scss

```diff
@@ -0,0 +1,260 @@
+@import url('https://fonts.googleapis.com/css2?family=Poppins:wght@500;700&display=swap');
+
+$main-color: #45a29e;
+$second-color: #66fcf1;
+$third-color:rgb(186, 124, 233);
+
+
+* {
+    font-family: 'Poppins', sans-serif;
+    margin: 0;
+    padding: 0;
+}
+main {
+	background: #1f2833;
+	min-height: 100vh;
+}
+.header {
+	text-align: center;
+	background-color:#0b0c10;
+	color:$main-color;
+}
+.container {
+	display: grid;
+	grid-template-columns: 200px 1fr;
+	gap: 20px;
+	padding: 10px;
+}
+.filter {
+	width: 200px;
+	height: 530px;
+	background-color:rgba(0, 0, 0, 0.7);
+	margin: 0;
+	box-shadow: 3px 3px 10px  #000;
+	position: fixed;
+	border-radius:5px;
+	&__age, &__name , &__sex {
```

Comment:
You can set one generic name for all filters instead of adding unique class names to each of them.